compiler/rustc_error_messages/src/lib.rs

```diff
@@ -76,18 +76,26 @@ fluent_messages! {
 
 pub use fluent_generated::{self as fluent, DEFAULT_LOCALE_RESOURCES};
 
+#[cfg(not(parallel_compiler))]
 pub type FluentBundle = fluent_bundle::bundle::FluentBundle<FluentResource, IntlLangMemoizer>;
```

Comment:
Could this unconditionally use `DynWrapper`?

Reply:
Done